clang/test/Preprocessor/sycl-esimd-force-stateless-mem.cpp

```diff
@@ -1,11 +1,13 @@
 /// This test checks that the macro __ESIMD_FORCE_STATELESS_MEM is automatically
-/// defined only if the option -fsycl-esimd-force-stateless-mem is used.
+/// defined by default with -fsycl, -fsycl-is-device or -fsycl-is-host.
```

Comment:
```suggestion
/// defined by default with -fsycl-is-device or -fsycl-is-host.
```
Just a nit.   *This* test being cc1 does not test what happens with -fsycl.

Reply:
This test was inspired by an existing test for SYCL: https://github.com/intel/llvm/blob/sycl/clang/test/Preprocessor/sycl-macro.cpp
cc1 needs to be only checked with -fsycl-is-device and -fsycl-is-host. -fsycl option seem not relevant to cc1.